oidc/clientassertion/doc.go

```diff
@@ -0,0 +1,13 @@
+package clientassertion
```

Comment:
What about a package doc string?

Reply:
good point, I'll do this tomorrow when I make an `example_test.go`